src/System.Runtime.InteropServices.RuntimeInformation/src/Architecture.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public enum Architecture
```

Comment:
Can this be defined in just one place? There is a copy in the ref... Also may be add a comment that this definition needs to be consistent with the native counterpart?